modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -24,11 +29,14 @@ For more information about how to install Helm, see the Helm installation docume
 ** Azure Kubernetes Service (AKS)
 * Not installed Istio in the cluster
 
-If you want to enable automatic sidecar injection, you must additionally have:
+If you want to enable automatic sidecar injection, you must additionally:
+
 * Install Kubernetes 1.12+ with admissionregistration API 
-*Tip:* To verify the installation, use the command `kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
++
+*Tip:* To verify the installation, use the command: 
++
+`kubectl api-versions | grep admissionregistration.k8s.io/v1beta1`.
```

Comment:
Consider making this a bullet item parallel to Install... and Add and list:

* Install Kubernetes...
* Verify the installation using...
* Configure kube-apiserver...
* Add and list

Reply:
Done! Thanks!

Reply:
@kbostrom and @priyaayyar, Shouldn't this be an ordered list (numbers) instead of bullets since there is a specific order?